src/headerData.json

```diff
@@ -175,5 +175,17 @@
         "$$hashKey":"object:26"
       }
     ]
+  },
+  "alert":{
+    "type":"blocker",
+    "message":"Para que puedas enviar tu próxima Campaña, antes necesitamos verificar el origen de tus Suscriptores.",
+    "button":{
+    "action":"validateSubscribersPopup",
```

Comment:
It is out of the scope of your task, but this kind of alert will not work in our webapp, right?

Reply:
Yes you are right. This button activates a popup.